agent.py

```diff
@@ -1222,11 +1220,11 @@ def spreadDisease(self, agent, disease):
         sugarscape.addDisease(disease, agent)
 
     def sortCellsByWealth(self, cells):
-        # Insertion sort of cells by wealth with range as a tiebreaker
+        # Insertion sort of cells by wealth in descending order with range as a tiebreaker
         i = 0
         while i < len(cells):
             j = i
-            while j > 0 and (cells[j - 1]["wealth"] > cells[j]["wealth"] or (cells[j - 1]["wealth"] == cells[j]["wealth"] and cells[j - 1]["range"] <= cells[j]["range"])):
+            while j > 0 and (cells[j - 1]["wealth"] < cells[j]["wealth"] or (cells[j - 1]["wealth"] == cells[j]["wealth"] and cells[j - 1]["range"] >= cells[j]["range"])):
```

Comment:
One last adjustment: the inverse of less than or equal to is strictly greater than. So, a quick swap of >= to just > should have this ready to merge. Plus, it'll add stability to the sort which I unintentionally broke in the first place.

Reply:
Good thinking, added in new commit.